communications.md

```diff
@@ -2,95 +2,62 @@
-
-Documentation - Transparency: Don't be embarrassed about asking "silly" questions. Try to always write your messages in a publically accessible (or at least a team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
-
-A few exceptions:
-
-- When you communicate credentials or secrets, use direct messages, or very well-controlled private channels.
-- When you talk about subjects that are irrelevant to work, use direct messages (but to be honest, wouldn't one of the social media/chat apps fit better?).
-
-#### - Internal & External Communication Tool: Gmail -
-
-Minimize Distractions: A busy email inbox is a contributing factor to interruptions. Instead of striving to achieve "Inbox Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. You may do the following:
-
-- Pin/star items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Optionally snooze items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Archive the rest.
-
-Opt-out: Use filters to avoid receiving emails sent to a shared email address that you are not interested in.
 
 #### - Project Management & Product Issue Documentation: Asana -
 
-Opt-in: Join projects that you are interested in or obligated to.
-
-Minimize Distractions: Similar to Slack, instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". Defer any information that is not work-related
-
-Documentation - Transparency: Provide as much detail as possible for other team members for context and opt-in to a task or issue if necessary.
-
-Opt-out: Leave projects that you are no longer interested in or obligated to.
-
-## Second line of Communication: Meeting
+Our Asana workspace is commonly used for project management e.g task allocation, task state (in progress or completed), and any other process we deem it fit to be used for (e.g Office Availability, calender scheduling, etc.)
+Opt-in to join projects/tasks that you are interested in or obligated to.
 
-If a meeting is necessary, budget the amount of time needed appropriately. Each meeting is the collective productivity value of all its participants. Consider each participant to be on average contributing $200/hr; it is always good to judge the need/value of each meeting against the economic value of the meeting (what other productive tasks could these participants be doing?).
```

Comment:
If the goal is to push this outside of dev, I agree we can remove some of the specifics to our workflow (knowledge-based, creative endeavours), like meetings having low priority. I like the sentiment of the first two paragraphs, but would stress that a team should outline and agree upon their codes of conduct for communication, not just person-person.

Definitely think that a lot of these points are relevant to how I work.

Reply:
we shouldn't expect every team outside dev just "take" what we have here, we are not aiming to set a company wide enforcement on some kind rule of communication, rather to show other team on how we do communication, they can take or try out whatever they think make sense in their line of work, make changes or improvements. we are showing them the thought process and reasoning behind why we have these guidelines, so they can arrive to their own way through similar process. the bottom line is they can understand our way of communication and know how to communicate to us.